examples/nas/spos/dataloader.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+import nvidia.dali.ops as ops
```

Comment:
why using this package?

Reply:
To accelerate the data loading by PyTorch.

Reply:
more detail? by accelerate what?

Reply:
ImageNet dataloading and augmentation is slow and inefficient. Running with PyTorch built-in dataloader induces bottleneck on CPU and memory. Using dali brings over 10x speedup on our workstation (4 GTX 1080 and a 12-core CPU).

Reply:
The main difference is to do data decoding and augmentation on GPU. This also brings some changes to the interface of dataloader.

Reply:
Got it and thx.

Reply:
better to mention this requirement in doc

Reply:
agreed, should offer a requirement.txt

Reply:
It's already mentioned in docs. DALI needs different installation command for cuda 9 and 10. Can't do them all in a `requirements.txt`: https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/installation.html

Reply:
can it execute by a sh script?